pkg/storage/storage_test.go

```diff
@@ -508,16 +508,16 @@ func TestStorageAPIs(t *testing.T) {
 							manifestBuf, err = json.Marshal(manifest)
 							So(err, ShouldBeNil)
 							digest := godigest.FromBytes(manifestBuf)
-							_, err = imgStore.PutImageManifest("test", digest.String(),
+							_, err = imgStore.PutImageManifest("test", "0.0.1",
```

Comment:
Why do we need these changes?

Reply:
this was before adding isBlobOlderThan(gc.delay) check, thanks for the heads up, I have reverted these changes.